### ISSUE TITLE
Where is the module "ilu_deepml"?

### ISSUE THREAD

**Issue by lyuchengcheng** [stes/compressed_sensing#1]
Hi, Steffen. Very thanks for your work. I want to implement your code, but I got a little problem.
In your code file, In[3] from ilu_deepml import tools. But I cannot find this module. Is it writed by yourself? And how can I find it?
Thank you very much!

**Comment by stes** [comment]
@lyuchengcheng Fair point, this is indeed an external module I will add in the coming weeks, stay tuned

**Comment by alfrednoble** [comment]
@lyuchengcheng @stes Hi Steffen, thanka a lot. I have also the exact same problem as @lyuchengcheng . I cannot find the module ilu_deepml in your files

**Comment by zhu12340008** [comment]
@stes@lyuchengcheng Hello, I have encountered the same problem as you. Have you found the ilu_deepml module? If you find it, I hope you can share it. Thank you very much.

**Comment by jorgebaccaq** [comment]
@stes @lyuchengcheng @zhu12340008 , Hello,  I have encountered the same problem as you. Have you found the ilu_deepml module? If you find it, I hope you can share it. Thank you very much.

**Comment by Bouderbal-Imene** [comment]
Hi @stes @zhu12340008 @jorgebaccaq @lyuchengcheng, please tell me where can I found the ilu_deepml module or with what can I remplace it

**Comment by Zero-Kiryu** [comment]
Hi @stes @lyuchengcheng @alfrednoble @zhu12340008 @jorgebaccaq @Bouderbal-Imene , I have encountered the same problem, could you please share the ilu_deepml module? Thanks~
